new/LIT6740PPWegat.xml

```diff
@@ -5,7 +5,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader>
         <fileDesc>
             <titleStmt>
-                <title xml:lang="en" xml:id="t1">Protective prayer in Gǝʿǝz and Amharic against stomachache (<foreign xml:lang="gez">wəgʿāt</foreign>
+                <title xml:lang="en" xml:id="t1">Protective prayer in Gǝʿǝz and Amharic against stomachache (<foreign xml:lang="gez">wəgāt</foreign>
```

Comment:
Sorry, is it wəgāt the proper form? I think I missed something. Dillmann gives ውግአት (https://betamasaheft.eu/Dillmann/lemma/L34927d16d21b476a8074e6637e1a0a2e). Perhaps wəgāt  is Amharic

Reply:
you're right, it's Amharic (but not so rare in scrolls, I think...not sure though). But since this is the form attested here, I would not like normalising it in the title. Should we just change the language here to Amharic?

Reply:
I would change the language to Amharic then!